configuration/scripts/cice.run.setup.csh

```diff
@@ -43,7 +41,9 @@ set ICE_RUNLOG_FILE = "cice.runlog.\${stamp}"
 #--------------------------------------------
 cd \${ICE_RUNDIR}
 
-setenv OMP_NUM_THREADS ${nthrds}
+setenv OMP_NUM_THREADS \${ICE_NTHRDS}
```

Comment:
On some systems, it's only via the job scheduler that you can specify the number of OpenMP threads, changing `OMP_NUM_THREADS` at runtime has no effect. I think it's the case at least for PBS Pro, the number of threads is set from the `ompthreads` directive in the select statement. Maybe that would be worth to mention in the documentation ? (maybe not, just thinking out loud...)

Reply:
I wasn't aware of that.  I guess that's part of setting up the port properly.